scripts/webpack/webpack.base.babel.js

```diff
@@ -15,17 +15,17 @@ process.env.REACT_WEBEX_VERSION = version;
 module.exports = (options, env) => {
   const packageJson = require('../../package.json');
   const plugins = [
-    new webpack.EnvironmentPlugin([
-      'NODE_ENV',
-      'WEBEX_CLIENT_ID',
-      'REACT_WEBEX_VERSION',
-      'WDM_SERVICE_URL',
-      'IDBROKER_BASE_URL',
-      'WEBEX_TEST_USERS_CONVERSATION_SERVICE_URL',
-      'WEBEX_CONVERSATION_DEFAULT_CLUSTER',
-      'FEDERATION',
-      'U2C_SERVICE_URL'
-    ]),
+    new webpack.EnvironmentPlugin({
+      NODE_ENV: "production",
```

Comment:
This change is required because when we run build:src using cross-env we specify node_env as production. If here we dont mention node_env as production. We get and error saying that there is a mismatch in node_env value

Reply:
Since this is in the Base config, won't it affect the dev config?

Reply:
If you look at what the command used to be, yarn build:all was using corss_env with node_env variable as production. So the dev config was also using NODE_ENV as production. And I dont think it affects anyything in the build.